src/uct/ib/ud/base/ud_iface.h

```diff
@@ -427,8 +426,11 @@ uct_ud_iface_check_grh(uct_ud_iface_t *iface, void *packet, int is_grh_present,
      *     - 0x0 : GRH - (RoCE v1.0)
      *     - 0x1 : IPv6 - (RoCE v1.5/v2.0)
      *     - 0x2 : IPv4 - (RoCE v1.5/v2.0)
+     * Some RoCE drivers fail to set the sl correctly so instead of relying
+     * on the sl the check uses the current address family to detect the
+     * correct gid length.
      */
-    gid_len = ((roce_pkt_type & UCT_IB_CQE_SL_PKTYPE_MASK) == 0x2) ?
```

Comment:
`UCT_IB_CQE_SL_PKTYPE_MASK` is not used anymore - can be deleted

Reply:
Removes.